_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -3451,6 +3466,154 @@ public static function activate_akismet() {
 		}
 	}
 
+	/**
+	 * Install a specific plugin.
+	 *
+	 * @since 8.9.0
+	 *
+	 * @param WP_REST_Request $request {
+	 *     Array of parameters received by request.
+	 *
+	 *     @type string $slug Plugin slug.
+	 * }
+	 *
+	 * @return WP_REST_Response A response indicating whether or not the installation was successful.
+	 */
+	public static function install_plugin( $request ) {
+		$plugin = stripslashes( $request['slug'] );
+
+		jetpack_require_lib( 'plugins' );
+		$result = Jetpack_Plugins::install_plugin( $plugin );
+
+		if ( is_wp_error( $result ) ) {
+			return rest_ensure_response(
+				array(
+					'code'    => 'failure',
+					'message' => esc_html(
+						sprintf(
+							/* translators: %1$s: plugin name. -- %2$s: error message. */
+							__( 'Unable to install %1$s: %2$s ', 'jetpack' ),
+							$plugin,
+							$result->get_error_message()
+						)
+					),
+				)
+			);
+		} else {
+			return rest_ensure_response(
+				array(
+					'code'    => 'success',
+					'message' => esc_html(
+						sprintf(
+							/* translators: placeholder is a plugin name. */
+							__( 'Installed %s', 'jetpack' ),
+							$plugin
+						)
+					),
+				)
+			);
+		}
+	}
+
+	/**
+	 * Activate a specific plugin.
+	 *
+	 * @since 8.9.0
+	 *
+	 * @param WP_REST_Request $request {
+	 *     Array of parameters received by request.
+	 *
+	 *     @type string $plugin Plugin long slug (slug/index-file)
+	 *     @type string $status Plugin status. We only support active in Jetpack.
+	 * }
+	 *
+	 * @return WP_REST_Response A response indicating whether or not the activation was successful.
```

Comment:
This can return a `WP_REST_Response` object or a `WP_Error` object, so the maybe something like this:

```suggestion
	 * @return WP_REST_Response|WP_Error A response object if the activation was successful, or a WP_Error object if the activation failed.
```